strongbox-web-core/src/test/java/org/carlspring/strongbox/controllers/layout/nuget/NugetArtifactControllerTest.java

```diff
@@ -205,11 +203,10 @@ public void testPackageCommonFlow(@NugetRepository(storageId = STORAGE_ID,
         final String packageVersion = "1.0.0";
 
         long packageSize = Files.size(packagePath);
-        byte[] packageContent = readPackageContent(packagePath);
 
         // Push
         String url = getContextBaseUrl() + "/storages/{storageId}/{repositoryId}/";
-        createPushRequest(packageContent).when()
+        createPushRequest(packagePath).when()
                                          .put(url, storageId, repositoryId)
```

Comment:
Cosmetic: Could you please re-indent the chained methods? Thanks! :)

Reply:
Fixed.